.github/workflows/tests.yml

```diff
@@ -19,7 +19,6 @@ jobs:
       fail-fast: false
       matrix:
         py-version:
-          - 3.6
           - 3.7
```

Comment:
So at the end also 3.7 works fine?
Instead, for 3.6 nothing to do?

Reply:
I have changed it back to where it was so that the required dependencies that h5py installation wants will be chosen. Python 3.6 should also work as we have it now so I will enable it and see what happens. If it is still failing we can disable parallel h5py from that version.

Reply:
Apparently not ^^ I will remove it for now.

Reply:
Ok, no problem!